### ISSUE TITLE
iOS app hangs with white screen

### ISSUE THREAD

**Issue by pysiak** [BlueWallet/BlueWallet#544]
I have a few readonly (xpub and zpub) wallets imported.
The app stopped to start properly last week or so. When I tap the app icon, it briefly show the bluewallet logo, then a spinning wheel for fraction of a second and then stays white as in the screenshot.
I don't think I have means to further debug or reset the app. Offloading and installing again didn't help.
Any ideas?
![F632DEE8-D844-4166-B825-46733DC4C68B](https://user-images.githubusercontent.com/2396137/61185772-546b5380-a65d-11e9-837f-8177273c4177.png)
Thanks

**Comment by Overtorment** [comment]
Have you tried to import wallets on other device? Or without vpn?

**Comment by pysiak** [comment]
Tried to open without VPN, on Wifi, on LTE. Same behaviour. That VPN is not really VPN, but DNSCloak for dnscrypt - turning it off doesn't fix it. Will try on another device later in the week and report.
Also tried the 4.2.0 beta with RBF stuff, didn't work either.
Would be nice to know what operation the app is doing.

Perhaps you could add a small icon, which after a tap, would reveal some checkpoints: connecting to X, loading Y, retrying Z, assert on xyz, etc....

**Comment by pysiak** [comment]
So, I imported xpub and zpubs in a different device and it was ok there. On the not-working-device I did a reinstall and reimport. It worked (network conditions/wifi/lte/vpn are the same)
I fear some conditions may make it to re-occur with the wallets I have. I will be monitoring.

Are you considering adding some kind of debug so that if it happens again I'd have something more verbose to report? When started, the bluewallet logo appears briefly and disappears, then a 'loading' animation appears and disappears, and then just stays blank.
